### PR TITLE
Removed the errorMessage prop in Select

### PR DESCRIPTION
**Pull request by robinpiets**
See #519

### CODE REVIEW

File: packages/asc-ui/src/components/Select/Select.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import { createRef, useEffect } from 'react'
+import { withKnobs, boolean, text } from '@storybook/addon-knobs'
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Select/Select.stories.mdx:2]: These aren't used, but you can make a useful story with these! Checkout `Accordion.stories.mdx`

File: packages/asc-ui/src/components/Select/Select.tsx [hunk 2]
```diff
@@ -12,15 +12,20 @@ const Select = React.forwardRef<
...
+      error,
       errorMessage,
       labelStyle,
-      errorStyle,
       ...otherProps
     },
     ref,
   ) => {
-    const error = errorProp || errorMessage
+    if (errorMessage) {
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Select/Select.tsx:22]: I think we can either show this warning, and still keep the `FormLabelStyle` (and don't mark it in the changelog as breaking, but deprecated.) OR we can remove this and consider it a breaking change

Comment (jonkoops) [reply packages/asc-ui/src/components/Select/Select.tsx:22]: Let's remove it entirely.

Comment (robinpiets) [reply packages/asc-ui/src/components/Select/Select.tsx:22]: 👍

File: packages/asc-ui/src/components/Select/Select.stories.mdx (hunk 1 above)
Comment (robinpiets) [reply packages/asc-ui/src/components/Select/Select.stories.mdx:2]: Cool! I removed them for now.

**Comment by jonkoops** [comment]
@robinpiets LGTM, can you squash your commits before merging?

**Comment by robinpiets** [comment]
> @robinpiets LGTM, can you squash your commits before merging?

Done!

**Pull request merged**